operations/mimir/autoscaling.libsonnet

```diff
@@ -11,6 +11,8 @@
     autoscaling_ruler_querier_min_replicas: error 'you must set autoscaling_ruler_querier_min_replicas in the _config',
     autoscaling_ruler_querier_max_replicas: error 'you must set autoscaling_ruler_querier_max_replicas in the _config',
     autoscaling_ruler_querier_cpu_target_utilization: 1,
+    autoscaling_ruler_querier_memory_target_utilization: 1,
+    autoscaling_ruler_querier_use_oom_trigger: false,
```

Comment:
Personally I don't think we need to have this behind a flag unless we consider it an experimental feature.

I would rather confirm that the OOM protection works on the queries and then roll it out to all of the autoscalers.

Reply:
Sounds good to me! I'll remove the flag and just change the query for all autoscalers. I'm also going to change this PR back to a draft until we have good data on using the new query for the rest of the autoscalers.